cmd/rabbitmq-setup/main.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2014 ISRG.  All rights reserved
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package main
+
+// This command does a one-time setup of the RabbitMQ exchange and the Activity
+// Monitor queue, suitable for setting up a dev environment or Travis.
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/codegangsta/cli"
+	"github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/streadway/amqp"
+
+	"github.com/letsencrypt/boulder/cmd"
+)
+
+// Constants for AMQP
+const (
+	monitorQueueName = "Monitor"
+	amqpExchange     = "boulder"
+	amqpExchangeType = "topic"
+	amqpInternal     = false
+	amqpDurable      = false
+	amqpDeleteUnused = false
+	amqpExclusive    = false
+	amqpNoWait       = false
+)
+
+func setup(c *cli.Context) {
```

Comment:
Are we sure we want to keep using this cli lib? It seems like a lot of complexity for not much gain.